diff --git a/cpp/open3d/t/geometry/kernel/IPPImage.cpp b/cpp/open3d/t/geometry/kernel/IPPImage.cpp
index 4c763f3..b7e2a7a 100644
--- a/cpp/open3d/t/geometry/kernel/IPPImage.cpp
+++ b/cpp/open3d/t/geometry/kernel/IPPImage.cpp
@@ -123,7 +123,7 @@ void Resize(const open3d::core::Tensor &src_im,
 
     auto it = type_dict.find(interp_type);
     if (it == type_dict.end()) {
-        utility::LogError("Unsupported interp type {}", interp_type);
+        utility::LogError("Unsupported interp type {}", static_cast<int>(interp_type));
     }
 
     try {
